webapp/packages/plugin-user-profile/src/UserProfileForm/UserForm.m.css

```diff
@@ -0,0 +1,67 @@
+.tabList {
+  position: relative;
+  flex-shrink: 0;
+  align-items: center;
+}
+
+.topBar {
+  composes: theme-border-color-background theme-background-secondary theme-text-on-secondary from global;
+  position: relative;
+  display: flex;
+  padding-top: 16px;
+
+  &:before {
+    content: '';
+    position: absolute;
+    bottom: 0;
+    width: 100%;
+    border-bottom: solid 2px;
+    border-color: inherit;
+  }
+}
+
+.topBarTabs {
+  flex: 1;
+}
+
+.topBarActions {
+  display: flex;
+  align-items: center;
+  padding: 0 24px;
+  gap: 16px;
+}
+
+/*Button:not(:first-child) {
+    margin-right: 24px;
+  }*/
+
+.statusMessage {
+  composes: theme-typography--caption from global;
+  height: 24px;
+  padding: 0 16px;
+  display: flex;
+  align-items: center;
+  gap: 8px;
+
+  & IconOrImage {
+    height: 24px;
+    width: 24px;
+  }
+}
+
+.flexBox {
```

Comment:
Probably we should give it a different name, .box or something like that.
Also we have a component with similar styles `<Container vertical overflow  />`, we could use this